Fix for skipping poll for active projects

itavero opened the pull request:
As mentioned in PR #24, we still experienced some issues with the previous change.
I've tested this on our environment and this does work without any problems.

casz commented on src/main/java/hudson/plugins/accurev/AccurevSCM.java:
```diff
@@ -402,13 +404,13 @@ public PollingResult compareRemoteRevisionWith(@Nonnull Job<?,?> project, @Nulla
-		if (activeProject == null) {												   
-            activeProject = project;
-        } else {
+		if (activeProject != null && activeProject.isBuilding()) {
```
can you fix the indent :)

itavero replied:
You are commenting on an old version I believe.

casz commented:
Did you fix it polling only on master?

itavero commented:
Yes, it works now.
Apparently my fix did not fix enough. ;)
It still reported not needing a workspace.
After I added some more logging, I finally found that this was the case.

It looks to me that the conditions that were combined with OR operators, where not correctly combined.
That's why I split them, so I could clearly see the decisions being made.
